boa/contracts/vyper/vyper_contract.py

```diff
@@ -462,6 +471,20 @@ def __repr__(self):
         return repr(self.dump())
 
 
+# data structure to represent the constants in a contract
+class ConstantsModel:
+    def __init__(self, compiler_data: CompilerData):
+        for v in compiler_data.annotated_vyper_module.get_children(VariableDecl):
+            if v.is_constant:
+                setattr(self, v.target.id, v.value.reduced().value)
```

Comment:
maybe `get_folded_value()` -- it is more strict since it raises an exception if there is no folded value (which would be either a bug in the compiler or an API change)